.ci/travis.sh

```diff
@@ -69,6 +71,69 @@ checkstyle-tester-diff-groovy-patch-only)
     -pc my_check.xml -p patch-branch -r ../.ci-temp/checkstyle -m single
   ;;
 
+checkstyle-tester-diff-groovy-regression-single)
+  # Check out lateset checkstyle from master
+  checkout_from https://github.com/checkstyle/checkstyle
+
+  # Run report from master branch of contribution
+  checkout_from https://github.com/checkstyle/contribution
+  cd .ci-temp/contribution/checkstyle-tester
+  sed -i'' 's/^guava/#guava/' projects-to-test-on.properties
+  sed -i'' 's/#checkstyle|/checkstyle|/' projects-to-test-on.properties
+  export MAVEN_OPTS="-Xmx2048m"
+  groovy ./diff.groovy --listOfProjects projects-to-test-on.properties \
+    -pc ../../../checkstyle-tester/diff-groovy-regression-config.xml \
```

Comment:
Whats the difference compared to https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/checks-nonjavadoc-error.xml ?

Reply:
Right, I basically copied that config and made it regression compatible.